cms/static/js/spec/views/active_video_upload_list_spec.js

```diff
@@ -89,6 +91,40 @@ define(
                 });
             });
 
+            describe('file size upload', function() {
+                it('blocks file uploads larger than the max file size', function() {
```

Comment:
What about `it('can not upload file larger than the max file size', function() {`?